package.json

```diff
@@ -155,6 +155,7 @@
     "globalthis": "^1.0.2",
     "hammerjs": "^2.0.8",
     "hash-wasm": "^4.4.1",
+    "headroom.js": "^0.12.0",
```

Comment:
Not a new addition, we already load this using `third_party.js`. We will remove this library from third_party once ajs is removed.

Reply:
Please add a TODO somewhere about this.

Reply:
Do we really need a TODO for this, Isn't it obvious that angularjs libraries will be removed along with angularjs after migration?

\cc @srijanreddy98

Reply:
I mean it is kinda obvious, but it is also something we can easily forget about. It would be different if it was something that we would need to do, but that is not the case here.

Reply:
done, created TODO #12855